src/SeleniumLibrary/__init__.py

```diff
@@ -42,133 +42,190 @@
+    Selenium's WebElement objects. This requires first getting a WebElement,
+    for example, by using the `Get WebElement` keyword.
+
+    | ${elem} =       | `Get WebElement` | id=example |
+    | `Click Element` | ${elem}          |            |
+
+    == Custom locators ==
+
+    If more complex lookups are required than what is provided through the
+    default locators, custom lookup strategies can be created. Using custom
+    locators is a two part process. First, create a keyword that returns
+    a WebElement that should be acted on:
+
+    | Custom Locator Strategy | [Arguments] | ${browser} | ${strategy} | ${tag} | ${constraints} |
+    |   | ${element}= | Execute Javascript | return window.document.getElementById('${criteria}'); |
+    |   | [Return] | ${element} |
+
+    This keyword is a reimplementation of the basic functionality of the
+    ``id`` locator where ``${browser}`` is a reference to a WebDriver
+    instance and ``${strategy}`` is name of the locator strategy. To use
+    this locator it must first be registered by using the
+    `Add Location Strategy` keyword:
+
+    | `Add Location Strategy` | custom | Custom Locator Strategy |
+
+    The first argument of `Add Location Strategy` specifies the name of
+    the strategy and it must be unique. After registering the strategy,
+    the usage is the same as with other locators:
+
+    | `Click Element` | custom:example |
+
+    See the `Add Location Strategy` keyword for more details.
+
+    == Locating tables ==
+
+    Locating tables when using table related keywords, such as
+    `Table Should Contain`, works differently than locating other elements.
+    The main differences are:
+
+    - The default strategy is to search only by ``id``, not by ``name``.
+    - Implicit XPath strategies using format ``//example`` is not supported.
+    - Only explicit strategies that are supported are ``xpath``, ``css``
+      and ``sizzle/jquery``.
+    - Spaces are not allowed around the separator between the strategy and
+      the value.
+
+    Examples:
+
+    | `Table Should Contain` | example                         | text | # Match based on ``id``.        |
+    | `Table Should Contain` | css=table.example               | text | # Match using CSS selector.     |
+    | `Table Should Contain` | xpath=//table/[@name="example"] | text | # Match using XPath expression. |
+
+    The plan is to uniform locating tables and locating other elements in
+    the future.
 
     = Timeouts =
 
-    There are several `Wait ...` keywords that take timeout as an
+    There are several ``Wait ...`` keywords that take timeout as an
     argument. All of these timeout arguments are optional. The timeout
-    used by all of them can be set globally using the
-    `Set Selenium Timeout` keyword. The same timeout also applies to
-    `Execute Async Javascript`.
+    used by all of them can be set globally using the `Set Selenium Timeout`
```

Comment:
Should we also say that timeout can be set in the library import?

Reply:
Good idea. Will add.